productsManagement/src/main/java/com/productsmanagement/controller/ProductManagementController.java

```diff
@@ -0,0 +1,30 @@
+package com.productsmanagement.controller;
+
+import com.productsmanagement.service.ProductManagementService;
+import com.skytouch.dto.ProductDTO;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
```

Comment:
This controller will be consumed by JSPs, are you sure that `@RestController` is the correct annotation you need here?

Reply:
I wanted to see if that was working using my postman, Im planning to change this to controller in order to use these services along JSP